frontend/src/metabase/search/components/SearchUserPicker/SearchUserPicker.tsx

```diff
@@ -52,17 +56,42 @@ export const SearchUserPicker = ({
   return (
     <SearchFilterPopoverWrapper
       isLoading={isLoading}
-      onApply={() => onChange(selectedUserId)}
+      onApply={() => onChange(selectedUserIds)}
     >
       <SearchUserPickerContainer p="sm" h="100%" spacing="xs">
-        <TextInput
-          size="md"
-          mb="sm"
-          placeholder={t`Search for users…`}
-          value={userFilter}
-          tabIndex={0}
-          onChange={event => setUserFilter(event.currentTarget.value)}
-        />
+        <SearchUserSelectBox spacing="xs" p="xs">
```

Comment:
Adds user chips to the selection box to allow for multiple users and reflect similar behavior to that of the user picker within the QueryBuilder